devops/automation/azure-pipelines.yml

```diff
@@ -189,65 +189,65 @@ jobs:
     continueOnError: true
     condition: and(succeededOrFailed(), eq('${{ parameters.RunningTests }}', true))
 
```

Comment:
Two options:

1. We just remove the entire job
2. Use an if statement.

Since it looks like you don't want to remove the code, do the following: 

1. Add a param at the top of the yaml:

```yaml
- name: uploadToolchain
   type: boolean
   default: false
```

Then, just before the job:
```yaml
- ${{ if eq(uploadToolchain, true) }}:
```
Add the job under the if with an increase indentation.